### ISSUE TITLE
Populating roles into profile from Firestore

### ISSUE THREAD

**Issue by raygadas** [prescottprue/react-redux-firebase#580]
**Do you want to request a *feature* or report a *bug*?**  
Bug

(If this is a *usage question*, please **do not post it here**—post it on [gitter](https://gitter.im/redux-firebase/Lobby). If this is not a “feature” or a “bug”, or the phrase “How do I...?” applies, then it's probably a usage question.)

**What is the current behavior?**
Populate feature doesn't work properly when trying to populate a firebase.profile from a firestore collection.

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem via [codesandbox](https://codesandbox.io/) or similar.**
![image](https://user-images.githubusercontent.com/11649786/49471523-bebc3b80-f7d2-11e8-88e1-19bacd19f966.png)

**What is the expected behavior?**
The field "role" under firebase.profile should be filled with the content from the "roles" collection. For instance, if a user has a role "admin", which is the key in my "roles" collection, it should populate the field with that content but instead I still get the "admin" string.

**Comment by prescottprue** [comment]
@raygadas Did you include `profileParamsToPopulate`? For real time database `populate` works with the `profile` by automatically attaching listeners when passed `profileParamsToPopulate` as [outline in the populate docs](http://react-redux-firebase.com/docs/recipes/populate.html#populate-profile-parameters) (since the listener is automatically managed). That said, I am not sure of whether I got the chance to add that for firestore profiles.

**Comment by raygadas** [comment]
@prescottprue I tried including it before but it gave me a warning on the console so I deleted it. I have now added it again, I keep getting the same warning and the profile isn't populated. I think I wrote/did everything properly, but maybe you can point out any mistake.   

Here's what I'm doing:
**Index.js**
![image](https://user-images.githubusercontent.com/11649786/49517382-b4954e00-f861-11e8-847f-3ba35be6c8aa.png)

**Dashboard.js**
![image](https://user-images.githubusercontent.com/11649786/49517483-f0c8ae80-f861-11e8-8449-5bffb4e591e1.png)
![image](https://user-images.githubusercontent.com/11649786/49517445-d4c50d00-f861-11e8-9758-7d5774dc399b.png)

**Firestore**
![image](https://user-images.githubusercontent.com/11649786/49517566-28cff180-f862-11e8-92a5-b7d0e2a39df4.png)
![image](https://user-images.githubusercontent.com/11649786/49517622-44d39300-f862-11e8-9d68-a2501b3d67d3.png)

**Console (with warning)**
![image](https://user-images.githubusercontent.com/11649786/49517695-67fe4280-f862-11e8-8917-74ec6ea7ba6f.png)

**Comment by prescottprue** [comment]
Yeah, you are doing everything right. That warning is letting you know that this feature doesn't exist yet for Firestore 😄. Good to hear there is interest in the feature though. I'll update when I have a better sense of when I can get to adding it.

**Comment by raygadas** [comment]
Sure, thank you. Any ideas of a how I could do it in the meantime?

**Comment by prescottprue** [comment]
You can query directly for the role by passing it as a prop and creating a query (either manually or through `firestoreConnect`). Then `populate` should have what it needs to create a populated profile.

I'll try to get to an example of what I mean some time this week, but adding support for the feature should be coming soon too.

**Comment by raygadas** [comment]
Sound good, I'll give it a try and will wait for your example too. Thanks for the help!

**Comment by sbtgE** [comment]
@raygadas @prescottprue can you tell me where I can find an example of how to do that? it would be very helpful!

**Comment by xdarkleonx** [comment]
I have same problem. Help me please https://stackoverflow.com/questions/58974610/react-redux-firebase-populate-doesn-t-work

**Comment by prescottprue** [comment]
@xdarkleonx this ticket is about populating the profile section of Firestore, which as mentioned, is not yet supported. It looks like in your stack overflow post though that you are trying to just populate an item in a list loaded from Real Time Database, not something on the users profile or from Firestore.

**Comment by xdarkleonx** [comment]
ok, my mistake

**Comment by waynebrantley** [comment]
@prescottprue Any thoughts on this feature being implemented.  Your library is really nice!

**Comment by glynnallen1704** [comment]
> @prescottprue Any thoughts on this feature being implemented. Your library is really nice!

Desperate for this feature too!

**Comment by jowo-io** [comment]
Would also love to see an example of a workaround for this for the time being!
